### DISCUSSION TITLE
Welcome to geopark-2023 Discussions!

### DISCUSSION THREAD

**Discussion by nquarder** [earthlab-education/geopark-2023#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rlopezmo1988** [comment]
working fine!

**Reply by nquarder** [reply]
got it @rlopezmo1988 !! Welcome :)

**Comment by nquarder** [comment]
Hello I'm in the discussion!

**Comment by fabioaco** [comment]
Here I am

-

**Reply by nquarder** [reply]
Welcome @fabioaco !
